core/src/main/java/org/apache/brooklyn/core/catalog/internal/CatalogBundleLoader.java

```diff
@@ -128,10 +128,12 @@ private TemporaryInternalScanResult scanForCatalogInternal(Bundle bundle, boolea
     }
 
     /**
-     * Remove the given items from the catalog.
+     * Remove the given item from the catalog.
      *
-     * @param item Catalog items to remove
+     * @param item Catalog item to remove
+     * @deprecated since 0.13.0 remove bundles
      */
+    @Deprecated
```

Comment:
Does not seem in the scope of this PR. Also not sure why deprecating it?

Reply:
Looks like code is not called, and it's in an internal package. Therefore I'm ahppy with it being deprecated (or even deleted immediately). Same for below.